src/style.css

```diff
@@ -0,0 +1,19 @@
+body {
+  margin: 0;
+}
+
+.block {
+  height: 100px;
+}
+
+.redBg {
```

Comment:
do not use camelCase in CSS, separate words by `-` or `_`

Also, red-bg is ok class for this task, but not for the future